tools/autograd/gen_variable_type.py

```diff
@@ -839,13 +899,12 @@ def emit_call(env):
             else:
                 unpacked_method_args = combined['unpacked_args'][1:]
                 base_type_call = CALL_DISPATCH_VIA_METHOD.substitute(
-                    combined, unpacked_method_args=unpacked_method_args)
+                    combined, var='self_', unpacked_method_args=unpacked_method_args)
             if not modifies_arguments and not returns_void:
-                rhs_value = wrap_output('tmp')
                 call = DISPATCH_TO_NON_VAR_TYPE_WITH_RETURN_VALUES.substitute(
-                    base_type_call=base_type_call,
-                    return_values=tie_return_values(),
-                    rhs_value=rhs_value)
+                    base_type_call=base_type_call)
+
+                call += wrap_output(tie_return_values(), 'tmp')
```

Comment:
What's going on here? I can't read the diff.

Reply:
Nothing really happens here, I'm just breaking on big CodeTemplate so that I can plugin the lambda function before returning. :P